torch/_dynamo/skipfiles.py

```diff
@@ -33,23 +33,6 @@
 import torch._export.constraints as _export_constraints
 import torch._inductor.test_operators
 
-
-try:
-    import torch._prims
-
-    # isort: split
-    # TODO: Hack to unblock simultaneous landing changes. Fix after https://github.com/pytorch/pytorch/pull/81088 lands
-    import torch._prims.utils
-    import torch._prims.wrappers
```

Comment:
Note these two modules don't exist, so `HAS_PRIMS_REFS=False` on `main`.

Since this has been the case for a long time I am assuming the extra `FILENAME_ALLOWLIST` isn't needed.